Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableAssetHandler.cpp

```diff
@@ -55,7 +54,6 @@ namespace AzFramework
         AZ::ObjectStream::FilterDescriptor filter(assetLoadFilterCB);
         if (AZ::Utils::LoadObjectFromStreamInPlace(*stream, *spawnable, nullptr /*SerializeContext*/, filter))
         {
-            SpawnableAssetEventsBus::Broadcast(&SpawnableAssetEventsBus::Events::OnPreparingSpawnable, *spawnable, asset.GetHint());
```

Comment:
should this bus be removed / deprecated or is this called elsewhere?

Reply:
This bus already existed and contains other events that are still being used, so it's still being called elsewhere.